pkg/util/meminfo.go

```diff
@@ -161,14 +165,22 @@ func readCgroupMemStat(memStatPath string) (int64, error) {
 
 // GetPodMemStatUsageBytes returns the pod's memory usage quantity (Byte)
 func GetPodMemStatUsageBytes(podCgroupDir string) (int64, error) {
```

Comment:
these sys util should be moved into koordlet pkg

Reply:
done